src/components/SideBar/ItemDrawer/ItemDrawer.tsx

```diff
@@ -99,25 +103,38 @@ export default class ItemDrawer extends React.PureComponent<Props, State> {
     return el
   }
 
-  getColumnCount(): number {
-    return Number(this.props.columnCount)
+  handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({ search: event.target.value })
+    this.handleSearchDebounced(event.target.value)
   }
 
-  handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
-    if (event.target.value.length > 0 && !this.state.isSearching) {
-      this.setState({ isSearching: true })
-    } else if (event.target.value.length === 0 && this.state.isSearching) {
-      this.setState({ isSearching: false })
+  handleOnCleanSearchClick = () => {
```

Comment:
Is this name ok? Why not call it `handleCleanSearch`?

Reply:
My bad!